tcms/testcases/forms.py

```diff
@@ -35,6 +36,14 @@ class Meta:
         queryset=Product.objects.all(),
         empty_label=None,
     )
+    setup_duration = forms.DurationField(
+        widget=DurationWidget(attrs={"id": "setup_duration"}),
```

Comment:
FTR I don't think you need to pass the id attribute because it is the same as the field name and that's the default value.

Reply:
Removing attrs actually fixes a crash that's happening.